cancer_ai/validator/model_runners/onnx_runner.py

```diff
@@ -45,8 +47,14 @@ def preprocess_data(self, X_test: List) -> List:
 
     async def run(self, X_test: List) -> List:
         import onnxruntime
+        try:
+            session = onnxruntime.InferenceSession(self.model_path)
+        except:
```

Comment:
print exception in the log as well

Reply:
```
except Exception as e:
bt.logging.error(f"Failed to run model: {e}")
```

sth like that

Reply:
Not sure what you mean.

Reply:
we need to log also what's the exception message, for example
https://github.com/safe-scan-ai/cancer-ai/blob/cdb3aa651e6418d8b797697ddf637062063adec7/cancer_ai/base/base_validator.py#L113